### ISSUE TITLE
Code Editor 2: You cannot switch a Document from GML Visual to GML Code, but you can vice-versa from the Editor frame

### ISSUE THREAD

**Issue by gurpreetsinghmatharoo** [YoYoGames/GameMaker-Bugs#5432]
### Description

The Editor Frame has a language button in GML Code mode:

![image](https://github.com/YoYoGames/YoYoStudio/assets/21360237/ebdae65a-03f9-4013-9513-3a038472f869)

If you change from GML to GML Visual here, that button is gone, and there is no way to switch the event back from the Editor Frame.

![image](https://github.com/YoYoGames/YoYoStudio/assets/21360237/98972bdc-f3b1-4c42-8327-cee292084483)

You can do that through the right-click menu, but from a UX perspective this will confuse users who expect to use the same button to function the same regardless of the Document (as it is part of the Editor Frame).

### Expected Change

The GML Visual editor should keep the same button to switch back to GML Code

### Steps To Reproduce

1. Start GameMaker, enable CE
2. Create an object, create a GML Code event
3. Switch to GML Visual, you can do that
4. Now you cannot switch back to code from the Editor itself
5. See the issue

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

 IDE v9.9.1.647  Runtime v9.9.1.766

### Which operating system(s) are you seeing the problem on?

Windows 11

### Are you running GameMaker from inside your Steam library?

None

### ui.log File

- [ ] I have attached my ui.log file

### Sample .yyz File

- [ ] I have attached a small sample project

**Comment by skud92** [comment]
I agree this would be expected UX, language dropdown wasn't in initial design.

We could add it to the bottom bar where the zoom controls are as a frame already exists
<img width="768" alt="image" src="https://github.com/YoYoGames/GameMaker-Bugs/assets/42850338/74bc6d30-25bd-435c-921d-6258b43d2400">

If this doesn't work a larger UX discussion should be had to redesign the framework to allow for the language select dropdown

Other suggested layouts - the background would have to use a frame like the right hand panel and sit over the content that already exists.
<img width="787" alt="image" src="https://github.com/YoYoGames/GameMaker-Bugs/assets/42850338/fe19713e-4a2f-405d-a8de-23155ef2643e">

<img width="1035" alt="image" src="https://github.com/YoYoGames/GameMaker-Bugs/assets/42850338/1e6d49de-3b85-4980-b417-cb6e484738d7">
